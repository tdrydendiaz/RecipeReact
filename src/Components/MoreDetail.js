import React, { Component } from 'react';


export class MoreDetail extends Component {

    render() {

        return (
            <div >
                <p>More Detail about Your Recipe</p>
            </div>

        );
    }
}

export default MoreDetail;
