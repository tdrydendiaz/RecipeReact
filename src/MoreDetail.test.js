import React from 'react';
import { shallow } from 'enzyme';
import MoreDetail from "./Components/MoreDetail";

it("clicks a button", () => {
  const wrapper = shallow(<MoreDetail title="A Title" />);

  expect(wrapper).toMatchSnapshot();

  wrapper.find('[name="Delete Recipe"]').simulate("click");

  expect(wrapper).toMatchSnapshot();
});
