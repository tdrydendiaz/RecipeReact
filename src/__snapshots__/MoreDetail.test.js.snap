// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clicks a button 1`] = `ShallowWrapper {}`;
